requests_cache/backends/base.py

```diff
@@ -11,6 +11,7 @@
 import hashlib
 from copy import copy
 from io import BytesIO
+from requests_cache.compat import urlencode, urlparse, urlunparse, parse_qs
```

Comment:
There is already ``from ..compat import...`` on line 18. Please, move this names there.